packages/component-base/test/virtualizer-item-height.test.js

```diff
@@ -181,3 +182,58 @@ describe('virtualizer - item height - sub-pixel', () => {
     expect(containerHeight).to.equal(Math.ceil(ITEM_HEIGHT));
   });
 });
+
+describe('virtualizer - item height - initial render', () => {
+  let virtualizer, elementsContainer, itemHeight, createElements;
+
+  beforeEach(() => {
+    const scrollTarget = fixtureSync(`
+      <div style="height: 300px;">
+        <div class="container"></div>
+      </div>
+    `);
+    const scrollContainer = scrollTarget.firstElementChild;
+    elementsContainer = scrollContainer;
+
+    createElements = sinon.spy((count) => Array.from({ length: count }, () => document.createElement('div')));
+
+    virtualizer = new Virtualizer({
+      createElements,
+      updateElement: (el, index) => {
+        el.style.width = '100%';
+        el.style.height = itemHeight;
+        el.textContent = `Item ${index}`;
+      },
+      scrollTarget,
+      scrollContainer,
+    });
+  });
+
+  it('should have the expected amount of physical elements: 100px item height', async () => {
```

Comment:
**nit:** The tests could be grouped into 2 test suites like this:

```js
describe('100px item height', () => {
  beforeEach(async () => {
    itemHeight = '100px';
    virtualizer.size = 100;
    await nextFrame();
  });

  it('should have the expected amount of physical elements', () => {
    // ...
  });
});
```

Reply:
Done